wcfsetup/install/files/lib/data/user/UserProfile.class.php

```diff
@@ -463,7 +469,9 @@ public function isAccessible($name) {
 	 * @return	boolean
 	 */
 	public function isProtected() {
-		return (!WCF::getSession()->getPermission('admin.general.canViewPrivateUserOptions') && !$this->isAccessible('canViewProfile') && $this->userID != WCF::getUser()->userID);
+		$data = ['result' => (!WCF::getSession()->getPermission('admin.general.canViewPrivateUserOptions') && !$this->isAccessible('canViewProfile') && $this->userID != WCF::getUser()->userID)];
```

Comment:
Insert a linebreak after the opening bracket and before the closing bracket for readability.